apps/server/src/modules/authentication/services/authentication.service.ts

```diff
@@ -40,18 +46,43 @@ export class AuthenticationService {
 		return account;
 	}
 
-	public async generateJwt(createJwtPayload: CreateJwtPayload): Promise<LoginDto> {
+	private async generateJwt(createJwtPayload: CreateJwtPayload, expiresIn?: number): Promise<string> {
```

Comment:
Contract check missed, only numbers greater then 0, or xxx are allowed.
Param expiresIn need to include the unit. Maybe expiresInMs?